Library/Homebrew/global.rb

```diff
@@ -5,6 +5,7 @@
 require "extend/ARGV"
 require "PATH"
 require "extend/string"
+require "vendor/backports/string"
```

Comment:
I'd rather we figure out how to fix that or just avoid the backport at this point, to be honest.